saleor/webhook/response_schemas/payment.py

```diff
@@ -122,3 +129,96 @@ class StoredPaymentMethodDeleteRequestedSchema(BaseModel):
             default=None,
         ),
     ]
+
+
+def clean_id(payment_method_id: str, info: ValidationInfo) -> str:
+    app: App | None = info.context.get("app", None) if info.context else None
+    if not app:
+        raise RuntimeError("Missing app in context")
+    return to_payment_app_id(app, payment_method_id)
+
+
+def clean_result(result: str):
+    return PaymentMethodTokenizationResult[result]
+
+
+class PaymentMethodTokenizationSchema(BaseModel):
+    id: Annotated[str, Field(description="ID of the payment method.")]
+    result: Annotated[  # type: ignore[name-defined]
+        Literal[
+            PaymentMethodTokenizationResult.SUCCESSFULLY_TOKENIZED.name,
+            PaymentMethodTokenizationResult.ADDITIONAL_ACTION_REQUIRED.name,
+        ],
+        Field(
+            description="Result of the payment method tokenization.",
+        ),
+        AfterValidator(clean_result),
+    ]
+    data: Annotated[
+        DefaultIfNone[JsonValue],
+        Field(
+            description="A data passes to the client.",
+            default=None,
+        ),
+    ]
+    error: Annotated[
```

Comment:
The error field is used by Saleor, only with specific result:
![image](https://github.com/user-attachments/assets/7787afba-d38b-48a2-a429-0a2359328acf)
IMO, we could be more strict here and remove error from "success" response